### PR TITLE
docs: remove named node version

### PR DESCRIPTION
**Pull request by StephenFluin**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

## PR Type
What kind of change does this PR introduce?

- [x] Documentation content changes

## Does this PR introduce a breaking change?

- [x] No

### CODE REVIEW

File: aio/content/guide/setup-local.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Before you begin, make sure your development environment includes `Node.js®` an
 {@a nodejs}
 ### Node.js
 
-Angular requires `Node.js` version 10.9.0 or later.
+Angular requires a [https://nodejs.org/en/about/releases/](current, active LTS, or maintenance) LTS of `Node.js`. See the `engines` key for the specific version requirements in our [package.json](https://unpkg.com/@angular/cli/package.json).
```
Comment (gkalpak) [inline aio/content/guide/setup-local.md]:
```suggestion
Angular requires a [current, active LTS, or maintenance](https://nodejs.org/about/releases/) version of `Node.js`. See the `engines` key for the specific version requirements in our [package.json](https://unpkg.com/@angular/cli/package.json).
```

**Review by gkalpak — APPROVED** [review]
👌

File: aio/content/guide/setup-local.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Before you begin, make sure your development environment includes `Node.js®` an
 {@a nodejs}
 ### Node.js
 
-Angular requires `Node.js` version 10.9.0 or later.
+Angular requires a [current, active LTS, or maintenance LTS](https://nodejs.org/en/about/releases/) version of `Node.js`. See the `engines` key for the specific version requirements in our [package.json](https://unpkg.com/@angular/cli/package.json).
```
Comment (gkalpak) [inline aio/content/guide/setup-local.md]: Nit: Removing the `en/` path segment from the URL allows detecting the user's language/locale.

```suggestion
Angular requires a [current, active LTS, or maintenance LTS](https://nodejs.org/about/releases/) version of `Node.js`. See the `engines` key for the specific version requirements in our [package.json](https://unpkg.com/@angular/cli/package.json).
```

**Review by gkalpak — APPROVED** [review]

**Review by IgorMinar — APPROVED** [review]
lgtm!

**Pull request closed without merging**